RUFAS/routines/manure/manure_nutrients/manure_nutrient_manager.py

```diff
@@ -0,0 +1,232 @@
+from __future__ import annotations
+
+import math
+
+from RUFAS.routines.manure.manure_nutrients.manure_nutrients import ManureNutrients
+from RUFAS.routines.manure.manure_nutrients.nutrient_request import NutrientRequest
+from RUFAS.routines.manure.manure_nutrients.nutrient_request_results import NutrientRequestResults
+
+
+class ManureNutrientManager:
+    def __init__(self):
+        """Initialize the manure nutrient manager."""
+
+        self._nutrients = ManureNutrients()
+
+    @property
+    def values(self) -> ManureNutrients:
+        """
+        Get the current nutrient values stored in the manager.
+
+        Returns
+        -------
+        ManureNutrients
+            The current nutrient values stored in the manager.
+
+        """
+        return self._nutrients
+
+    def add_nutrients(self, nutrients: ManureNutrients) -> None:
+        """
+        Add nutrients to the manager from the manure module.
+
+        Parameters
+        ----------
+        nutrients : ManureNutrients
+            The nutrients to be added to the manager.
+
+        Returns
+        -------
+        None
+
+        """
+        self._nutrients += nutrients
+
+    def request_nutrients(self, request: NutrientRequest) -> NutrientRequestResults | None:
+        """
+        Request nutrients from the crop and soil module.
+
+        Parameters
+        ----------
+        request : NutrientRequest
+            The request for nutrients.
+
+        Returns
+        -------
+        NutrientRequestResults
```

Comment:
```suggestion
        NutrientRequestResults | None
```
Also I think the case when None is returned should be described